### DISCUSSION TITLE
[Question]: Multiple events export

### DISCUSSION THREAD

**Discussion by kei81131** [blakeblackshear/frigate#11757] [Ask A Question] [answered]
### What is your question:

I want to save all dog events from these days separately
Is there an easy way to do this, thank you

**Comment by NickM-27** [comment, marked as answer]
in 0.14 you can right click the items that you want to export

**Reply by kei81131** [reply]
thanks so much, its user friendly
would like to know if it is possible to export multiple events at the same time?
(when I select 2 or more events at review page, there is no more export option)

btw, export is not working on (dev-36ae42a) or (dev-53fa64f), i can't find that events at export page, although it returned successfully exported
on (dev-ea0292b), export works great
